Xamarin.Forms.Platform.UAP/SliderRenderer.cs

```diff
@@ -115,6 +120,14 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateThumbColor();
 			else if (e.PropertyName == Slider.ThumbImageProperty.PropertyName)
 				UpdateThumbImage();
+			else if (e.PropertyName == Specifics.StepFrequencyProperty.PropertyName)
+				UpdateStepFrequency();
+		}
+
+		void UpdateStepFrequency()
+		{
+			if (Element.IsSet(Specifics.StepFrequencyProperty))
+				Control.StepFrequency = Element.OnThisPlatform().GetStepFrequency();
```

Comment:
This needs to update `Control.SmallChange` as well.

Reply:
Ideally, the logic in `OnElementChanged` would be moved to `UpdateStepFrequency` and `UpdateStepFrequency` would be called by `OnElementChanged`. 

Regardless, the logic in `UpdateStepFrequency` should handle the situation where step frequency is set back to `-1d` (in case the user wants to revert to the default behavior).